docs/how-to/no-backup.rst

```diff
@@ -0,0 +1,24 @@
+==========================================
+Run upgrades without cloud database backup
```

Comment:
For backup, since it applies to both `plan` and `upgrade`, I prefer keeping it outside `Upgrade a cloud`. But I will change its name so that it's clear that it can not only be used under upgrade.